examples/integration_testing.rs

```diff
@@ -146,181 +149,3 @@ where D: Database {
-        );
-    }
-
-    #[test]
-    fn non_shared_server_with_different_database() {
-        let server = {
-            let bots = vec!["bors".into(), "homu".into(), "highfive".into()];
-            let server = super::start_server("127.0.0.1:0", bots).unwrap();
-            Server::new(server)
-        };
-
-        let mut response = server.get("/users");
-
-        let json: Value = serde_json::from_str(&response.body()).unwrap();
-
-        assert_eq!(json["users"].as_array().unwrap().len(), 3);
-        assert_eq!(response.status, StatusCode::Ok);
-        assert_eq!(
-            response.headers.get::<header::ContentType>(),
-            Some(&header::ContentType::json())
-        );
-    }
-
-    mod support {
-        use hyper::client::{Client, Response as HyperResponse};
-        use nickel::ListeningServer;
-
-        use std::net::SocketAddr;
-
-        pub trait Body {
-            fn body(self) -> String;
-        }
-
-        impl<'a> Body for &'a mut HyperResponse {
-            fn body(self) -> String {
-                use std::io::Read;
-                let mut body = String::new();
-                self.read_to_string(&mut body).expect("Failed to read body of Response");
-                println!("Read body: {}", body);
-                body
-            }
-        }
-
-        /// An example wrapper type to make testing more readable
-        pub struct Server(SocketAddr);
-        impl Server {
-            pub fn new(server: ListeningServer) -> Server {
-                let wrapped = Server(server.socket());
-
-                // detaching is important otherwise it would block the test threads.
-                server.detach();
-
-                wrapped
-            }
-
-            pub fn get(&self, path: &str) -> HyperResponse {
-                let url = self.url_for(path);
-                Client::new().get(&url).send().unwrap()
-            }
-
-            pub fn post(&self, path: &str, body: &str) -> HyperResponse {
-                let url = self.url_for(path);
-                Client::new().post(&url).body(body).send().unwrap()
-            }
-
-            pub fn url_for(&self, path: &str) -> String {
-                format!("http://{}{}", self.0, path)
-            }
-        }
-
-        lazy_static! {
-            /// This is a shared instance of the server between all the tests
-            pub static ref STATIC_SERVER: Server = {
-                let server = super::super::start_server("127.0.0.1:0", vec![]).unwrap();
-                Server::new(server)
-            };
-        }
-
-        /// Example of a free function version of `get` which uses the shared server
-        pub fn get(path: &str) -> HyperResponse {
-            STATIC_SERVER.get(path)
-        }
-
-        /// Example of a free function version of `post` which uses the shared server
-        pub fn post(path: &str, body: &str) -> HyperResponse {
-            STATIC_SERVER.post(path, body)
-        }
-    }
-}
```

Comment:
I assue this is just nuking a bunch of tests that don't work.  Are they worth reimplementing/have they been reimplemented?  Not a blocker, just a question.